File: Admin2-Backend/src/Admin2.Domain/Models/Conta.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Admin2.Domain.Models
{
    public class Conta
    {
        public Conta()
        {
            TipoConta = new TipoConta();
            Jogos = new List<Compra>();
        }

        public int Id { get; set; }
        public string Email { get; set; }
        public string Senha { get; set; }
        public string IdOnline { get; set; }
        public TipoConta TipoConta { get; set; }
        public DateTime DataNascimento { get; set; }
        public bool Status { get; set; }
        public bool Ativa { get; set; }
        public DateTime DataDesativacao { get; set; }

        public List<Compra> Jogos { get; set; }
        public List<CodigosConta> Codigos { get; set; }
    }
}

File: Admin2-Frontend/src/app/conta/conta.ts
import * as moment from 'moment';

import { Compra } from '../compra/compra';

export class TipoConta {
    public id: number;
    public descricao: string;
}

export class CodigosConta {
    public id: number;
    public descricao: string;
    public status: boolean;
    public contaId: number;
}

export class Conta {

    constructor() {
        this.tipoConta = new TipoConta();
        this.jogos = new Array<Compra>();
        this.codigos = new Array<CodigosConta>();
    }

    public id: number;
    public email: string;
    public senha: string;
    public idOnline: string;
    public tipoConta: TipoConta;
    public dataNascimento: moment.Moment;
    public status: boolean;
    public ativa: boolean;
    public dataDesativacao: moment.Moment;

    public jogos: Array<Compra>;
    public codigos: Array<CodigosConta>;
}

export class JogosConta {
    public id: number;
    public contaId: number;
    public jogoId: number;
}

export class ContaFilter {
    public text: string;
}